### PR TITLE
Changes the bool parser to detect some more values

### PR DESCRIPTION
**Pull request by sd-lueckel**

### CODE REVIEW

File: src/Service/BoolParser.php [hunk 1]
```diff
@@ -16,7 +16,12 @@ class BoolParser implements BoolParserInterface
      */
     public function parse($value): bool
     {
-        $normalizedValue = \trim(\strtolower($value));
-        return \in_array($normalizedValue, ['yes', 'on', 'true', '1']);
+        if (\is_bool($value)) {
+            return $value;
+        }
+        if (\is_string($value)) {
```
Comment (michz) [inline src/Service/BoolParser.php]: Did you change the codestyle rules?
I personally prefer more newlines:
```
        if (\is_bool($value)) {
            return $value;
        }

        if (\is_string($value)) {
            $value = \trim(\strtolower($value));
        }

        return \in_array($value, ['yes', 'on', 'true', '1', 1]);
```

Comment (sd-lueckel) [reply src/Service/BoolParser.php]: No did not change anything on the codestyle rules.

Comment (michz) [reply src/Service/BoolParser.php]: I thought we forced that a long time ago. But I guess this made problems with block ending at the end of blocks or so....

Comment (sd-lueckel) [reply src/Service/BoolParser.php]: Yes that is possible

**Pull request merged**